### PR TITLE
Improve makemessages command to support no pot creation date

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: chamber/management/commands/makemessages.py [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
 
 
+def _remove_pot_creation_date(pofile):
+    if os.path.exists(pofile):
+        with open(pofile, "rb+") as f:
+            lines = f.readlines()
+            f.seek(0)
+            for line in lines:
+                if not line.startswith(b'"POT-Creation-Date: '):
+                    f.write(line)
```
Comment (matllubos) [inline chamber/management/commands/makemessages.py]: remove creation date from po file

Comment (rubickcz) [reply chamber/management/commands/makemessages.py]: OMG, i didnt know you must do it so manually..

Comment (Formulka) [reply chamber/management/commands/makemessages.py]: how is this not built in?

Comment (Formulka) [reply chamber/management/commands/makemessages.py]: Wouldn't this end up with the last line being in the file twice? the original and the one written over the second to last? You are writing to the original file from the start.

Comment (matllubos) [reply chamber/management/commands/makemessages.py]: yes gettext has no parameter for it

no for this reason there is truncate. Truncate removes next lines from the cursor

Comment (Formulka) [reply chamber/management/commands/makemessages.py]: oh ok

**Pull request merged**